vllm/model_executor/models/dbrx.py

```diff
@@ -291,7 +295,7 @@ def __init__(
             cache_config,
             quant_config,
             prefix=f"{prefix}.norm_attn_norm")
-        self.ffn = DbrxMoE(config, quant_config)
+        self.ffn = DbrxMoE(config, quant_config, prefix=f"{prefix}.moe")
```

Comment:
Think this needs to be ffn per https://huggingface.co/databricks/dbrx-instruct?show_file_info=model.safetensors.index.json
```suggestion
        self.ffn = DbrxMoE(config, quant_config, prefix=f"{prefix}.ffn")
```